puppet/modules/slave/files/pbuilder_F60addforemanrepo

```diff
@@ -0,0 +1,95 @@
+	d3P7KH209aRTu04N4k/b4OZhNUq1Nu45k6UCcza5G5E+tJ7UvambHFzg+tsRbU84
+	12nspYLffs+88WiSaBxLISvROhhN97xGoDVkbryC1H1+CfKDTYfJPV/ngK6nGkHo
+	dd8t1mj4bYqUBPdO2HyZgCjk1+M5whn5NqvlwgxCgP3gfssZeYA1yLiWk80behfZ
+	wuKHw4cfWEqMv3SOaazmXTrM9hAFxEpyYq+L/olP0cjOeCPqh7BHIBcQhTOfHMa1
+	wfWxWMq8tY+TuPBCMcEXSZCS68CBuy6vnE+qzokI53N4RxJLMsddohzJKHrI+RRP
+	YOsWypn+QFdfDiz0abC8GdIUx/U42gm4JiVBUh+6qUQDYcsPcC7AfFBmXwARAQAB
+	tD5Gb3JlbWFuIEF1dG9tYXRpYyBTaWduaW5nIEtleSAoMjAxNikgPHBhY2thZ2Vz
+	QHRoZWZvcmVtYW4ub3JnPokCPgQTAQIAKAIbAwYLCQgHAwIGFQgCCQoLBBYCAwEC
+	HgECF4AFAmBLaU0FCQslOnQACgkQb4YAuVYyePaOiRAAqBtJ/eeA33sWoAqlH4CN
+	dyC9P5icGnUxo1hYQlIaSh7w59p74ro3iqpwZH151CXET0zTXfICRQjufa1JeA9F
+	RGq/5O8lOD22j4axRZ3MhYn/jwvhjEjmlasCVNigc7VMaWgzRpweWF/oX2alGF71
+	JpDaHUfHIba4DzuNv6PD+H7GEl9ZmMnqhJF5PoCAVj1d+J9z+IXuZMqAWU5h91b3
+	lJQq/Xfpi5ReykrsdmYOxiveJdZtAP2gXC0Sq98hDt7BhSoE8QJlRY9jBjgRPyrY
+	cguO9KaTnCTn5XytJ+3ZBjSYVsVPxVYAur06qvEeIa3cp2UTfDShvdGGn6O6zf7n
+	jAJh1iPe/91WhdKv5/67txFb6mdN/4fXRpWUfETbFdffwNBsENS4E8WqTWLPBOw+
+	9rMlyoeSluI04es8Edc/Itkfu19Z1i2/YDLrThhX4o3dj1oAM0V3MxwOMw1S5hm+
+	UJ6ClMiV1EiHyBAK65pKn6y1IxTJbHdoxNDqBrRYeSCXXVFPxPRofyPAj8YIAGd3
+	ijlnY1PJyMhnGnMzFDQ8Vp3IUdkxteWa+DrcHe5NoLvl2wtSQ4TdSv/niQpVWddo
+	izEji5VcRcP3AmKWvcH4UOQdLWH3NsHAqKurk39KnLOh+F5iy5X5rWn7T7ue8YWl
+	+7s2SpY1JLlegiP4aEZ6RRm5Ag0EVwdiWQEQALJUNPo0bqmCsdPqp03ZlLQmuGNl
+	ua7jJzgvd2vCZrCW7ucFD3rxE+MkpKzE+6t8XslnUwAqW8TUG7i2QKg2JivS2ZU0
+	2Fj2i8/fOHp1mWLUJIpRT8UmW3AiMosN2K5vsCO9euwepMIrLnv3+3kKxu5i/ayy
+	nCkSo0BX2ETXvnflm1MO38DhHXIaOmMAUmVVJjsNVkX7UQSDLxAqiSCzPGGSWYiW
+	dsnxvvu/YKU0qjXEKfJVYb+74ozUILqgkLzl0bCvoYjWotqBIlIsY2MawqAxDgVo
+	ySkjhN0GHLCYRJ9v27Xc/O1x7kPqqmdq2+Mk9DFkkrdDOK/9NPnar0DXIXJVsqr5
+	pgJhm8/hUdxhXpVKKC+R6ONNNCZk3xxmHhjnY4A4LXRaDASq5Om8MxyyKT0cgroF
+	Q2xuU8plaUcIKM0B4qGEqdlDE+ZwsiN0WAmW/R3Hwg0sfIiDRr8PcYMReRK8RzR9
+	ej6AHuLKWmirigfH4lUmsA9sylf4q1dmPA1kNv5l1BnDNmnx0Wlq9qTe50LNQh59
+	NPLf1qqyCHwGDNQX/OrwYDRojA7dNrwGy2NGWBfhbt8T1XbAfqo5b7rae0g1pRUN
+	pJDMfGqlk/kXtrrmJSIuQOrwGn9KaJCk2DKX/STPY16G9neaQ6f1sOJJ24RskOhO
+	C6LJCGcDPPoscsVXABEBAAGJAiUEGAECAA8CGwwFAmBLacAFCQslOucACgkQb4YA
+	uVYyePYdWg//UKWo1a+2m3EOscOQYtTBH6rNfSIecgnWkCQuGnZuODK1c3vxGQL6
+	xkY9+1zMjLYUVAjG2h3Hd7WMU5agvemyTZf1gbdwSgVdt1y5NVWsBPKXoyn2vbm8
+	kE/gECP6+8gJSatWI931e3R+m93HqN3cCGw1WUVXuDA65TsgPuoDxAf1VoB1Ewti
+	Qa4Ni8lKzRQn3qc4LsHEn2XfVmebR7cWAtGCLbHCpqE3ZPRi1RnqWJ73NsLX2eq4
+	gVoqwchrqbOf1OICPr0vlkICTG8ydcy/3urziJdYQivEXK5USd27c5kK8p5YU6It
+	DOGPHe045uwpChLRyslY6qYm0WlhBgWIAFr8dzhAyOhuWz3I6Iw43DACLa+co+WR
+	FZ33FqYTbl+F5x+x983alzXG0+8/eJ0fdcEOPj+C/GbJaKwXjpQk3l7OpUwLIQNy
+	nFVRXYaD1g2i6pHgWhhqFlWqo6fHrbD2Hzm9HCNS5eWiB/JGFZKyGBpBMh22/k6O
+	bIZ8SVgErE+AwvAE7HCnr+Pun/Iblrlb6fLC6J4WHumvmAIPLznIViUekC0jvEym
+	kNo8dY3LUn8NlQJ/qtBZDNf/OvP5LjiHPJdLlTyc+eZDGIszkPwLO/rjUfJGaD20
+	1vYLOIp+VREV4ODtMQfkLv7W+ynhzAgOZJBiJWU0CwLbnWza4D+yIIc=
+	=Fyc2
+	-----END PGP PUBLIC KEY BLOCK-----
+	-----BEGIN PGP PUBLIC KEY BLOCK-----
+	mQINBGD6Y/UBEADOhGOD74lvHdUAew1cHjAUTbopfoXvFvo5jXs4ra3IblnzPEOJ
```

Comment:
you lost a newline between the "begin block" and the actual content, and I know from painful experience, that `apt` doesn't like that (gpg does)